src/api/responses/phaseshift.ts

```diff
@@ -10,11 +10,11 @@ const phaseShiftTypeName = "PhaseShift";
 // phase for each account in the Erdstall system.
 @jsonObject
 export class PhaseShift extends ErdstallObject {
-	@jsonBigIntMember() current_epoch: bigint;
+	@jsonBigIntMember({ name: "current_epoch" }) currentEpoch: bigint;
```

Comment:
We discussed to change this to camelCase in the JSON API. Waiting for the Core PR :)

Reply:
🔀